docs/Development/Optimizely.md

```diff
@@ -0,0 +1,13 @@
+[Optimizely.com](https://optimizely.com/) is a A/B testing tool that helps you optimize your site's conversion rate. In order to use this module, you'll need an Optimizely account. A Free 30 day trial account is available.
+
+From your account page at [https://app.optimizely.com/v2/accountsettings/account/overview](https://app.optimizely.com/v2/accountsettings/account/overview)
+Get the Account ID
+![Account ID field depicted](../../media/optimizely_account_id.png?raw=true)
```

Comment:
I have made some changes, it seems the content will need to be in the repo to make them visible but these should work per https://github.com/mark-anders/relative-image-url example.